snyk/cli
# chore: Remove test summary message [CFG-1782]

#### What does this PR do?

- Removes the test summary message with the project counts for:
    1. Tested projects.
    2. Projects with vulnerabilities.
    3. Projects that were failed to be tested.

#### Where should the reviewer start?

- `src/cli/commands/test/iac/index.ts`

#### How should this be manually tested?

- Test a directory without failing projects, and verify that the summary message and the user tips both aren't displayed.
- Test a directory with failing projects, and verify that the summary message isn't displayed, but the user tips are displayed.

#### Any background context you want to provide?

In the original output, whenever multiple results are generated for a test, we display a short summary message with project counts for how many projects were tested, how many projects had issues, and how many projects failed to be tested.

In the new test output, these counts are displayed as part of the test summary section, and as such, with this message, we’d be displaying this information twice. As such, we’d like to remove this summary message in the new test output.

However, we’d like to continue displaying the user tip below it for debugging projects that failed to be tested.

#### What are the relevant tickets?

- [CFG-1782](https://snyksec.atlassian.net/browse/CFG-1782)

#### Screenshots

- With invalid results:
    ![image](https://user-images.githubusercontent.com/46415136/163781616-23371892-235f-4b40-8ce2-4d5e9ed355ca.png)
    
- Without invalid results:
    ![image](https://user-images.githubusercontent.com/46415136/163781710-7208ae87-a969-42b8-890d-a10d136ec621.png)
